CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ project(visualization.milkdrop2)
 cmake_minimum_required(VERSION 2.6)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR})
+set(CMAKE_ASM_NASM_COMPILER ${ADDON_DEPENDS_PATH}/bin/nasm.exe)
```

Comment:
`find_program`

Reply:
I wasn't sure if find_program would set the correct variable as that one is used by enable_language(ASM_NASM)